src/player_compchecks.c

```diff
@@ -41,6 +41,7 @@
 #include "power_hand.h"
 #include "gui_soundmsgs.h"
 #include "game_legacy.h"
+#include "cursor_tag.h"
 
 #include "keeperfx.hpp"
```

Comment:
did you check if it still needs #include "keeperfx.hpp" after the refactor?

Reply:
Apparently, it doesn't.